assets/css/refills/_badges.scss

```diff
@@ -0,0 +1,21 @@
+$small-font-size: 0.75em;
+$badges: (
+  "alert": #fff6bf,
+  "default": #999,
+  "error": #fbe3e4,
+  "notice": #e5edf8,
+  "success": #e6efc2,
+) !default;
+
+@each $badge-type, $color in $badges {
+  .badge-#{$badge-type} {
+    background-color: $color;
```

Comment:
Properties should be ordered display, padding, font-size, line-height, color, background-color, border-radius

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)